packages/elements-react/src/theme/default/components/card/auth-methods.tsx

```diff
@@ -19,23 +18,24 @@ export function DefaultAuthMethodListItem({
   switch (group) {
     case "passkey":
       Icon = passkey
-      title = "Passkey"
-      description = "Use your fingerprint or face to sign in."
+      title = "Passkey (recommended)"
+      description = "Use your devices for fringerprint or face recognition"
```

Comment:
These will need translations, but I don't think we have that for the other elements specific messages, yet. So let's just keep it in mind.